conu/backend/origin/registry.py

```diff
@@ -17,10 +17,14 @@
 """
 This are the methods helpful while working with OpenShift internal docker registry
 """
+import logging
 
 from conu.backend.origin.constants import INTERNAL_REGISTRY_PORT
 from conu.backend.docker.backend import DockerBackend
 from conu.utils import get_oc_api_token
+import conu.backend.origin.backend
```

Comment:
it would be a better user experience to do `from conu import OpenshiftBackend`.

Reply:
+1